src/Umbraco.Web.UI.Client/src/views/components/notifications/umb-notifications.html

```diff
@@ -9,12 +9,15 @@
             <div ng-if="notification.view">
                 <div ng-include="notification.view"></div>
             </div>
-
-            <div ng-if="notification.headline">
-                <a ng-href="{{notification.url}}" target="_blank">
+            <div ng-if="notification.headline" ng-switch on="{{notification}}">
+                <a ng-href="{{notification.url}}" ng-switch-when="{{notification.url != null}}" target="_blank">

```

Comment:
Maybe add a check on notification url not is empty send via `notificationService`
https://24days.in/umbraco-cms/2014/umbraco-angular-tips/

I think something like this might work.
```
ng-switch-when="{{notification.url && notification.url.trim() !== ''}}"
```

Reply:
Thanks :) 👍

Reply:
I'm wondering if this has worked for anyone else? I've been struggling to get the `url` set for a notification, (this is on v8.5.5).

re: `ng-switch`  - it's my understanding that `ng-switch-when` only works with literal values, rather than expressions. I'm a bit confused how this should work. 🤔 

I'm happy to send a PR over, but just checking if it did work for others first?